src/utils.ts

```diff
@@ -0,0 +1,21 @@
+export function parseCountFromStatus(status: string): number {
+	if (!/in\sthis\scall/.test(status)) {
+		return 0;
+	}
+	if (/is\sin\sthis\scall/.test(status)) {
+		return 1;
+	}
+	const names = status
+		.replace(/\sare\sin\sthis\scall/, '')
+		.split(/,\s|\sand\s/);
+	const moreMatch = names[names.length - 1].match(/(\d+)\smore/);
+	if (moreMatch !== null) {
+		return names.length - 1 + parseInt(moreMatch[1], 10);
+	} else {
+		return names.length;
+	}
+}
```

Comment:
Gathered Meet status data that this was based off of:

const one = 'Christina Wang is in this call';
const two = 'Christina Wang and Christina Wang are in this call';
const three = 'Christina Wang, Christina Wang and Christina Wang are in this call';
const four = 'Christina Wang, Christina Wang, Christina Wang and Christina Wang are in this call';
const five = 'Christina Wang, Christina Wang, Christina Wang and 2 more are in this call';
const six = 'Christina Wang, Christina Wang, Christina Wang and 3 more are in this call';
const seven = 'Christina Wang, Christina Wang, Christina Wang and 4 more are in this call';
const many = 'Christina Wang, Christina Wang, Christina Wang and 14 more are in this call';